internal/controller/article.go

```diff
@@ -0,0 +1,77 @@
+package controller
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+	"news/internal/entity"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+type articleInf interface {
```

Comment:
`articleInf` sounds odd to me. What does `inf` mean? 🤔

Reply:
fixed